### ISSUE TITLE
Are there Kitti seq05 and 15 ground truth labels available?

### ISSUE THREAD

**Issue by zeroAska** [shichaoy/semantic_3d_mapping#16]
I noticed in the code `grid_sensor/src/main_stereo_imgs.cpp`, it writes the reprojected images for evaluation. But where can we find the ground truth? The website of Kundu14 seems to be down. 
Thnaks!

**Comment by shichaoy** [comment]
hi thanks for the interest. I uploaded a folder ```preprocess_data/gt_label/``` containing the processed ground truth images used in our paper.

**Comment by zeroAska** [comment]
Hi Shichay, thanks for the reply! I noticed that the ground truth of kitti05 contains 40 images, indexed every 20 frames, from 000000 to 000800. But in the `data_kitti` folder, the rgb imgs are provided from 000000- 000020. This means that we can evaluate only on 000000 and 000020. If we want to use all the 40 ground truth images, do we have to use cnn and SLIC to generate data up to 000800?

**Comment by shichaoy** [comment]
yes, you need to process all of them (more than 800, then project to those image ID). I just uploaded 20 images, as they are too large.
